app-emulation/wine-staging/wine-staging-6.12-r1.ebuild

```diff
@@ -0,0 +1,666 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
Comments go for changes to vanilla and staging

Reply:
Done.